coffee-shop-service/src/index.js

```diff
@@ -130,22 +106,14 @@ const resolvers = {
     }
   },
   Order: {
-    customer(order) {
-      return lookupCustomer(order.customerId)
-    },
-  },
-  Customer: {
-    orders(customer) {
-      return getOrders().filter( order => order.customerId===customer.id)
-    },
+    customer: (order) => {
+      return {id: order.customerId}
+    }
```

Comment:
for demo -- since I already carry the piece of data I am interested in I figured I would directly attach it. 

I can fake that data as well, but that seemed confusing, what I need is another apollo client operation which gets customer data when that field is resolving.

**Would you rather it be faked with some pretty `Date.now()` info or do you want me to leverage the apollo client to make a request to the Java BE for this customer data?**

Reply:
Since the React App doesn't ask for it it doesn't at present make much of a difference